course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -250,6 +253,16 @@ def test_include_deleted_programs(self):
         )
         self.assertEqual(serializer.data, self.get_expected_data(self.course, self.request))
 
+    def test_filter_edx_org_short_name(self):
+        """
+        Filter by edx organization.
+        """
+        serializer = self.serializer_class.prefetch_queryset(
+            edx_org_short_name=self.edx_org_short_name,
+            partner=self.partner
+        )
+        assert serializer.data == self.get_expected_data(self.course2, self.request)
```

Comment:
`course2` ? shouldn't it be `course_2`? why tests didn't failed with this change?

Reply:
Done!
Because tests are not running, I'm writing them blindly as they are failing already some library is broken I tried to fix it but no luck.

Reply:
We need to make sure that the test are running (at least the ones we are writing) and passing otherwise there is no point in writing tests.